CANTroller2/src/devices.h

```diff
@@ -704,24 +390,61 @@ class PressureSensor : public AnalogSensor<int32_t, float> {
             human.set_limits(from_native(native.get_min()), from_native(native.get_max()));
             set_native(min_adc);
             set_can_source(ControllerMode::PIN, true);
-            if (pot_arg) {
-                enable_pot_input(pot_arg);
-            }
+            set_can_source(ControllerMode::POT, true);
         }
         PressureSensor() = delete;
 
         void setup() {
             set_pin(_pin, INPUT);
             set_source(ControllerMode::PIN);
         }
-    
-        // Convert units from base numerical value to disp units:  val_native = m-factor*val_numeric + offset  -or-  val_native = m-factor/val_numeric + offset  where m-factor, b-offset, invert are set here
-        void set_convert(float arg_m_factor, float arg_b_offset, bool arg_invert) {
-            _m_factor = arg_m_factor;
-            _b_offset = arg_b_offset;
-            _invert = arg_invert;
-            set_native(native.get());
+};
+
+// TODO: add description
```

Comment:
// BrakePositionSensor represents a linear position sensor 
// for measuring brake position (TODO which position? pad? pedal?)
// Extends AnalogSensor for handling analog pin reading and conversion.

Reply:
ty